pack/tcu/wos_encounter.json

```diff
@@ -0,0 +1,811 @@
+	},
+	{
+		"clues": 0,
+		"code": "05175",
+		"back_link": "05175b",
+		"encounter_code": "the_wages_of_sin",
+		"encounter_position": 15,
+		"faction_code": "mythos",
+		"illustrator": "Dual Brush Studios",
+		"name": "Chapel Attic",
+		"pack_code": "wos",
+		"position": 175,
+		"quantity": 1,
+		"shroud": 4,
+		"text": "<b>Forced</b> - After you draw a non-weakness card from your deck while at Chapel Attic: Place that card facedown beneath Chapel Attic, out of play.",
+		"type_code": "location"
+	},
+	{
+		"clues": null,
+		"code": "05176b",
+		"encounter_code": "the_wages_of_sin",
+		"encounter_position": 16,
+		"faction_code": "mythos",
+		"illustrator": "Dual Brush Studios",
+		"hidden": true,
+		"name": "Chapel Attic",
+		"pack_code": "wos",
+		"position": 176,
+		"quantity": 1,
+		"shroud": 8,
+		"text": "While investigating Chapel Attic, add the number of cards in your hand to your skill value for the investigation.\n<b>Haunted</b> - Discard a random card from your hand (2 cards instead of you have 5 or more cards in hand).",
+		"traits": "Spectral.",
+		"type_code": "location"
+	},
+	{
+		"clues": 0,
+		"code": "05176",
+		"back_link": "05176b",
+		"encounter_code": "the_wages_of_sin",
+		"encounter_position": 16,
+		"faction_code": "mythos",
+		"illustrator": "Dual Brush Studios",
+		"name": "Chapel Attic",
+		"pack_code": "wos",
+		"position": 176,
+		"quantity": 1,
+		"shroud": 8,
+		"text": "While investigating Chapel Attic, add the number of cards in your hand to your skill value for the investigation.",
+		"type_code": "location"
+	},
+	{
+		"code": "05177",
+		"cost": 0,
+		"deck_limit": 2,
+		"encounter_code": "the_wages_of_sin",
+		"encounter_position": 17,
+		"faction_code": "neutral",
+		"illustrator": "Apterus",
+		"name": "Spectral Web",
+		"pack_code": "wos",
+		"position": 177,
+		"quantity": 4,
+		"text": "[action] Investigators at your location spend 1-3 clues, as a group: <b>Fight.</b> Use this ability only to attack a [[Geist]] enemy. You may choose to use your [willpower] instead of your [combat] for this attack. You get +X skill value and deal +X damage for this attack, where X is the amount of clues spent to trigger this ability.",
+		"traits": "Spell.",
+		"type_code": "asset"
+	},
+	{
+		"code": "05178a",
+		"back_link": "01578b",
+		"encounter_code": "the_wages_of_sin",
+		"encounter_position": 21,
+		"enemy_damage": 1,
+		"enemy_evade": 3,
+		"enemy_fight": 4,
+		"enemy_horror": 1,
+		"faction_code": "mythos",
+		"health": 2,
+		"illustrator": "Stanislav Dikolenko",
+		"name": "Heretic",
+		"pack_code": "wos",
+		"position": 178,
+		"quantity": 1,
+		"text": "Heretic gets +2[per_investigator] health.\nWhile Heretic is at a non-[[Spectral]] location, it gains aloof and cannot be engaged or damaged.\n[free] Spend 1 clue: <b>Parley.</b> Look at Heretic's other side <i>(without resolving its text)</i>.\n<b>Forced</b> - After Heretic is defeated: Flip it over and resolve the text on its other side.",
```

Comment:
please add a space between '+2' and '[per_investigator]'. Please do this for each heretic.